src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -639,35 +639,73 @@ def make_files(self, entry):
             files.append(translated_file)
         return files
 
+    def make_translated_specimen(self, specimen):
```

Comment:
"translate" sounds rather generic and doesn't make clear what the difference is between this function and `make_specimens`. Both translate but only one has that term in its name.

```suggestion
    def make_specimen(self, specimen):
```

Focusing on plural vs singular is more expressive.